project_status.py

```diff
@@ -298,25 +295,355 @@ def main(args):
                     sample_entry['MedianInsertSize']
                 )
 
+def get_low_coverage(project):
+    # run same script again, to parse the output
+    # because all the functions above don't return anything, but just print the result
```

Comment:
ok... then make the functions return something and not rerun the script... in this way this function will be dependent by the output, if I want to add a coulmn I might need to change thawk command used. 
Moreover, everything that is printed is stored in `samples ` so by parsing samples you can fetch all information you need